src/HawksearchVue.js

```diff
@@ -771,18 +774,6 @@ class HawksearchVue {
         return configIndexName || urlIndexName || "";
     }
 
-    static getTabField(store) {
```

Comment:
We can leave this helper method although it is not used currently. Basically, it is a good way to fetch a tab facet an a reference for fetching other types of facets.